projects/plugins/jetpack/changelog/add-additional-menu-items-to-domain-only-sidebar

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Action Bar: introduce new module (currently only scaffolding).
```

Comment:
This changelog isn't exactly clear to me. Looks like what you had before was more descriptive?

Reply:
Fixed!